### PR TITLE
[tests-only][full-ci] add test to check copy activities of destination file and folder

### PR DESCRIPTION
**Pull request by prashant-gurung899**
## Description
This PR adds tests to check activities of destination file and folder after copying a file into a folder.
``` feature
Scenario: check activities of destination file and folder after copying a file into it
Scenario: check activities of destination file and folder after copying a file by renaming the destination file
Scenario: check activities of destination file after copying a file into same folder by renaming the destination file
```

## Related Issue

- Part of: https://github.com/owncloud/ocis/issues/9543
- https://github.com/owncloud/ocis/issues/9543#issuecomment-2437022281

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiActivities/activities.feature [hunk 1]
```diff
@@ -3208,3 +3208,544 @@ Feature: check activities
...
+                }
+              ]
+            }
+          }
+        }
+      }
+      """
+
+
+  Scenario: check activities of destination file and folder after copying a file by renaming the destination file
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiActivities/activities.feature:3435]:
```suggestion
  Scenario: check activities of destination resource after copying a file by renaming the destination file
```

File: tests/acceptance/features/apiActivities/activities.feature [hunk 2]
```diff
@@ -3208,3 +3208,544 @@ Feature: check activities
     And user "Alice" has uploaded file with content "updated ownCloud test text file" to "textfile.txt"
     When user "Brian" tries to list the activities of file "textfile.txt" from space "Personal" owned by user "Alice" using the Graph API
     Then the HTTP status code should be "403"
+
+
+  Scenario: check activities of destination file and folder after copying a file into it
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiActivities/activities.feature:3213]: can't we use resource word rather than file and folder

Comment (prashant-gurung899) [reply tests/acceptance/features/apiActivities/activities.feature:3213]: Yeah, we can. I was just trying to be specific with file and folder.

File: tests/acceptance/features/apiActivities/activities.feature [hunk 3]
```diff
@@ -3208,3 +3208,544 @@ Feature: check activities
     And user "Alice" has uploaded file with content "updated ownCloud test text file" to "textfile.txt"
     When user "Brian" tries to list the activities of file "textfile.txt" from space "Personal" owned by user "Alice" using the Graph API
     Then the HTTP status code should be "403"
+
+
+  Scenario: check activities of destination resource after copying a file into it
```
Comment (saw-jan) [inline tests/acceptance/features/apiActivities/activities.feature:3213]:
```suggestion
  Scenario: check activities of destination resources after copying a file
```

File: tests/acceptance/features/apiActivities/activities.feature [hunk 4]
```diff
@@ -3208,3 +3208,544 @@ Feature: check activities
...
+                }
+              ]
+            }
+          }
+        }
+      }
+      """
+
+
+  Scenario: check activities of destination resource after copying a file by renaming the destination file
```
Comment (saw-jan) [inline tests/acceptance/features/apiActivities/activities.feature:3435]:
```suggestion
  Scenario: check activities of destination resources after copying a file by renaming the destination file
```

File: tests/acceptance/features/apiActivities/activities.feature [hunk 5]
```diff
@@ -3208,3 +3208,544 @@ Feature: check activities
...
+                }
+              ]
+            }
+          }
+        }
+      }
+      """
+
+
+  Scenario: check activities of destination file after copying a file into same folder by renaming the destination file
```
Comment (saw-jan) [inline tests/acceptance/features/apiActivities/activities.feature:3657]: also check the activities of parent folder
```suggestion
  Scenario: check activities of destination resources after copying a file into same folder by renaming the destination file
```

**Review by amrita-shrestha — COMMENTED** [review]
lgtm

**Review by saw-jan — APPROVED** [review]
:+1:

**Pull request merged**